docs/_snippets/parameters-in-story.md

```diff
@@ -11,9 +11,10 @@ export default meta;
 type Story = StoryObj<Button>;
 
 export const Primary: Story = {
+  // 👇 Story-level parameters
   parameters: {
     backgrounds: {
-      values: [
+      options: [
         { name: 'red', value: '#f00' },
         { name: 'green', value: '#0f0' },
         { name: 'blue', value: '#00f' },
```

Comment:
It's not just the name that [changed](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#new-parameters-format-for-addon-backgrounds), it's the shape too:

```suggestion
      options: [
        red: { name: 'Red', value: '#f00' },
        green: { name: 'Green', value: '#0f0' },
        blue: { name: 'Blue', value: '#00f' },
```